### PR TITLE
Redesign reputaton summary

### PR DESCRIPTION
**Pull request by dkent600**
**Note** Small numbers ( > 0 ) switch to exponential notation when the number of leading zeroes reaches 3 (.000).  Hovering over a number shows a tooltip showing the entire number.

image

### CODE REVIEW

File: src/organizations/dashboard.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export class Dashboard {
   private totalReputationAvailable: BigNumber;
   private _loading: boolean = false;
   private initialized: boolean = false;
-
+  private exponentialAtForRep = [-4, 20];
```
Comment (orenyodfat) [inline src/organizations/dashboard.ts]: what does it means ?

File: src/organizations/dashboard.ts [hunk 2]
```diff
@@ -481,58 +475,50 @@ export class Dashboard {
...
       totalReputationAvailable = totalReputationAvailable.add(contractRepReward);
 
-      if (earnedRep.gt(0)) {
-        redeemables.push({
-          amount: earnedRep,
-          what: 'locked ETH',
-        });
-      }
+      redeemables.push({
+        amount: this.web3Service.fromWei(earnedRep),
```
Comment (orenyodfat) [inline src/organizations/dashboard.ts]: why fromWei ?

File: src/organizations/dashboard.ts (hunk 1 above)
Comment (dkent600) [reply src/organizations/dashboard.ts]: documented [here](https://github.com/daostack/dutchx-bootstrapper-ui/blob/c2ec505b275f538555275da090b3908ddd8f8095/src/services/numberService.ts#L32).

Added a comment in dahboard, [here](https://github.com/daostack/dutchx-bootstrapper-ui/blob/a507bcbbf1eb3dd06f78b7614c1f8fccee03d274/src/organizations/dashboard.ts#L82).

File: package.json [hunk 3]
```diff
@@ -120,7 +120,7 @@
     "webpack-dev-server": "^3.1.14"
   },
   "dependencies": {
-    "@daostack/arc.js": "0.0.0-alpha.111",
+    "@daostack/arc.js": "0.0.0-alpha.112",
```
Comment (orenyodfat) [inline package.json]: is this taged ?

File: src/organizations/dashboard.ts [hunk 4]
```diff
@@ -114,8 +118,9 @@ export class Dashboard {
       }],
     ]);
 
+  // tslint:disable: align
```
Comment (orenyodfat) [inline src/organizations/dashboard.ts]: please open a  new pr for this linting modification ...

Comment (dkent600) [reply src/organizations/dashboard.ts]: changed method parameter commas

**Pull request closed without merging**